Vagrantfile

```diff
@@ -107,10 +108,11 @@ Vagrant.configure("2") do |config|
     config.vm.provision :shell, :path => "shell/InstallChocolatey.ps1"
     config.vm.provision :shell, :path => "shell/NotifyGuiAppsOfEnvironmentChanges.ps1"
   else
-    config.vm.provision :shell, :path => "shell/PrepareWindows.ps1", :powershell_elevated_interactive => true
-    config.vm.provision :shell, :path => "shell/InstallNet4.ps1", :powershell_elevated_interactive => true
-    config.vm.provision :shell, :path => "shell/InstallChocolatey.ps1", :powershell_elevated_interactive => true
-    config.vm.provision :shell, :path => "shell/NotifyGuiAppsOfEnvironmentChanges.ps1", :powershell_elevated_interactive => true
+    config.vm.provision :shell, :path => "shell/PrepareWindows.ps1", privileged: false
+    config.vm.provision :shell, :path => "shell/InstallNet4.ps1", privileged: false
+    config.vm.provision :shell, :path => "shell/InstallChocolatey.ps1", privileged: false
+    config.vm.provision :shell, :path => "shell/NotifyGuiAppsOfEnvironmentChanges.ps1",privileged: false
+    config.vm.provision :shell, :path => "shell/InstallToolchain.ps1", privileged: false
```

Comment:
Why non-privileged? It needs to succeed, which is highly likely to need to run with admin on the VM. Not sure I understand this change

Reply:
See https://github.com/hashicorp/vagrant/issues/9138 for the ugly details. Looks like a change in Windows Server 2016  Build 14393.rs1_release.170917-1700.

Happy to change it if you know another work-around for Windows Server 2016. If you read the ticket the upstream maintainer of the packer scripts reports that this specific build of Windows Server 2016 has the problem.

So can work around it or wait(?) for Microsoft to change thing back?